pkg/leakybucket/overflows.go

```diff
@@ -14,9 +15,85 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/antonmedv/expr"
+	"github.com/antonmedv/expr/vm"
 	"github.com/crowdsecurity/crowdsec/pkg/exprhelpers"
 )
 
+const (
+	maxContextValueLen = 4000
+)
+
+func truncate(values []string) (string, error) {
```

Comment:
IIRC we have a similar behavior for storing meta in DB, maybe we can use the same code to do it ?